client/src/pages/Profile.js

```diff
@@ -9,14 +9,39 @@ import { withPageContext } from '../components/pageContext';
 
 import './profile.css';
 
+const styles = {
+  pageContent: {
+    display: 'flex',
+  },
+  headerContainer: {
+    display: 'flex',
+    flexDirection: 'row',
+  },
+  header: {
+    display: 'inline',
+    margin: '10px',
+    padding: '20px',
+    '&:hover': {
+      cursor: 'pointer',
+      color: '#E4E8EA',
+      textShadow: '1px 1px #B2B1B7',
```

Comment:
I know i mentioned in (another?) pr about the color change

Reply:
Yeah this should be changed in my edit project page. I had the idea that the edit project page would be merged before this branch